packages/p2p/src/pages/buy-sell/buy-sell-header.jsx

```diff
@@ -70,15 +70,15 @@ const BuySellHeader = ({ table_type }) => {
 
     return (
         <div
-            className={classNames('buy-sell__header', {
-                'buy-sell__header-position-static': !!buy_sell_store.api_error_message,
+            className={classNames('buy-sell-header', {
+                'buy-sell-header-position-static': !!buy_sell_store.api_error_message,
             })}
         >
-            <div className='buy-sell__header-container'>
+            <div className='buy-sell-header-container'>
```

Comment:
same here, i think it should follow component__child-component
```suggestion
            <div className='buy-sell-header__container'>
```